### PR TITLE
Force lookup

### PR DESCRIPTION
**Pull request by Babymcnerd**
Hey i would like to add my new changes to the force calculations, i added a lookup "curve" three functions given the speed.

### CODE REVIEW

File: src/car.rs [hunk 1]
```diff
@@ -111,7 +131,7 @@ fn calculate_tire_acceleration_and_braking_forces(
 }
 
 fn turn_tires(keys: Res<Input<KeyCode>>, mut tires: Query<(&mut Transform, &Tire)>) {
-    let turning_radius = PI / 5.0;
+    let turning_radius = 0.296706;
```
Comment (lukemcneil) [inline src/car.rs:134]: How'd you choose this number

File: src/car.rs [hunk 2]
```diff
@@ -145,7 +165,7 @@ fn calculate_tire_turning_forces(
             let desired_velocity_change = -steering_velocity * tire_grip_strength;
             let desired_acceleration = desired_velocity_change;
             ev_add_force_to_car.send(AddForceToCar {
-                force: steering_direction * desired_acceleration * 10.0,
+                force: steering_direction * desired_acceleration * car_transform.2 .0.mass,
```
Comment (lukemcneil) [inline src/car.rs]: Is there an extra space here

File: src/car.rs (hunk 1 above)
Comment (Babymcnerd) [reply src/car.rs:134]: Its the turning radius of the F1 car my roommate is building, so its a realistic number.

File: src/car.rs (hunk 2 above)
Comment (Babymcnerd) [reply src/car.rs]: yes there was

**Review by lukemcneil — APPROVED** [review]
Feel free to merge. Do you think doing logs will ever be too slow? Could also just make those parts of the graph linear

**Pull request merged**